<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FourierWindow</class>
 <widget class="QMainWindow" name="FourierWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>545</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>531</width>
         <height>504</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="Fourier" name="fourier" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>535</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDerp">
    <property name="title">
     <string>Fourier</string>
    </property>
    <addaction name="action_restore"/>
    <addaction name="action_resize"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_save_as"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>Filters</string>
    </property>
    <addaction name="action_high_pass"/>
    <addaction name="action_low_pass"/>
    <addaction name="action_band_pass"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDerp"/>
   <addaction name="menuFilter"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_high_pass">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/vector/icons/high_pass.png</normaloff>:/vector/icons/high_pass.png</iconset>
   </property>
   <property name="text">
    <string>High pass</string>
   </property>
  </action>
  <action name="action_low_pass">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/vector/icons/low_pass.png</normaloff>:/vector/icons/low_pass.png</iconset>
   </property>
   <property name="text">
    <string>Low pass</string>
   </property>
  </action>
  <action name="action_band_pass">
   <property name="icon">
    <iconset>
     <normalon>:/vector/icons/band_pass.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Band pass</string>
   </property>
  </action>
  <action name="action_restore">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/vector/icons/restore.png</normaloff>:/vector/icons/restore.png</iconset>
   </property>
   <property name="text">
    <string>Restore</string>
   </property>
  </action>
  <action name="action_update">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="action_resize">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/vector/icons/resize.png</normaloff>:/vector/icons/resize.png</iconset>
   </property>
   <property name="text">
    <string>Resize</string>
   </property>
  </action>
  <action name="action_save_as">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/vector/icons/save_file.png</normaloff>:/vector/icons/save_file.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="action_band_pass_2">
   <property name="text">
    <string>Band pass</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Fourier</class>
   <extends>QWidget</extends>
   <header location="global">ui.widgets.fourier</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/icons.qrc"/>
 </resources>
 <connections/>
</ui>
